frontend/src/pages/GraphPF/components/groupExpanded.tsx

```diff
@@ -61,6 +63,7 @@ type BaseGroupExpandedProps = {
   labelIconClass?: string; // Icon to show in label
   labelIcon?: string;
   labelIconPadding?: number;
+  isFocused?: boolean;
```

Comment:
```suggestion
  // Customizations 
  isFocused?: boolean;
  isHighlighted?: boolean;
  IsUnhighlighted?: boolean;
```

Please update that comment at the top in the same way you did for Node.tsx.  Also, we should probably add these props to Group.tsx so that StyleGroup.tsx is setting real props.

Reply:
Unresolving because I think you missed this comment.  I think we need to add the props and a comment to `Group.tsx`.  Also, we need to update the comment here in `GroupExpanded.tsx`.

Reply:
I don't see any changes to `group.tsx` :-)